src/appleseed/renderer/modeling/light/sunlight.cpp

```diff
@@ -418,6 +437,9 @@ namespace
                 + disk_radius * p[0] * basis.get_tangent_u()
                 + disk_radius * p[1] * basis.get_tangent_v();
 
+            if (target_point != Vector3d(0.0))
+                outgoing = normalize(target_point - position);
```

Comment:
That won't work because (0,0,0) is otherwise a perfectrly valid target point. We probably need to find a better mechanism here.